play/src/pusher/services/SocketManager.ts

```diff
@@ -1162,27 +1162,49 @@ export class SocketManager implements ZoneEventListener {
                 );
             })
             .catch((error) => {
-                debug(`ApiController => embeddableUrl : ${url} ${error.cause}`);
-                client.send(
-                    ServerToClientMessage.encode({
-                        message: {
-                            $case: "answerMessage",
-                            answerMessage: {
-                                id: queryMessage.id,
-                                answer: {
-                                    $case: "embeddableWebsiteAnswer",
-                                    embeddableWebsiteAnswer: {
-                                        url,
-                                        state: false,
-                                        embeddable: false,
-                                        message: "URL is not reachable",
+                if (isAxiosError(error) && error.response?.status === 999) {
+                    client.send(
+                        ServerToClientMessage.encode({
+                            message: {
+                                $case: "answerMessage",
+                                answerMessage: {
+                                    id: queryMessage.id,
+                                    answer: {
+                                        $case: "embeddableWebsiteAnswer",
+                                        embeddableWebsiteAnswer: {
+                                            url,
+                                            state: true,
+                                            embeddable: false,
+                                        },
```

Comment:
According to https://http.dev/999, Linked returns a 999 based on the user agent.
I guess it denies access to Axios.

Instead of putting this exception, we could try to send a user agent that matches the user agent of the client?